BLSQ/iaso
# POLIO-855: remove PPE

The field about PPE in Preparedness tab should be removed since it was 

Related JIRA tickets : POLIO-855

## Self proofreading checklist


## Changes

- removed PPE in parser.py

## How to test

- Go to polio
- Create a campaign with a round still in the future, and a scope (any scope)
- Save
- open the campaign again
- Go to preparedness tab
- click "generate spreadsheet"
- Once the spreadsheet is generated, click the button to open it
- In the sheet, go to a region tab and fill in some data
- Go back to iaso
- click "Refresh preparedness data"
- It should refresh

## Print screen / video

coming soon
